java.java

```diff
@@ -0,0 +1,11 @@
+// package descrition
```

Comment:
Typo inside comment in token 'descrition'. Possible corrections:description (90),descrition (0)